src/brainglobe_napari/atlas_download_dialog.py

```diff
@@ -0,0 +1,40 @@
+from bg_atlasapi.list_atlases import get_all_atlases_lastversions
+from qtpy.QtWidgets import (
+    QDialog,
+    QHBoxLayout,
+    QLabel,
+    QPushButton,
+    QVBoxLayout,
+)
+
+
+class AtlasDownloadDialog(QDialog):
```

Comment:
Is this necessary? Would [what we use in `brainreg-segment`](https://github.com/brainglobe/brainreg-segment/blob/34789347f3a908243dd991ff6120f7b3f232968f/brainreg_segment/layout/utils.py#L4) work? It's a bit simpler.

Reply:
These things could also go into the utils package maybe? Or even a specific brainglobe-napari-utils?

Reply:
Both good ideas, created https://github.com/brainglobe/brainglobe-utils/issues/3